skcosmo/feature_selection/voronoi_fps.py

```diff
@@ -0,0 +1,229 @@
+import numpy as np
+from time import time
+from .simple_fps import FPS
+import numbers
+
+
+class VoronoiFPS(FPS):
+    """
+    In FPS, points are selected based upon their Hausdorff distance to
+    previous selections, i.e. the minimum distance between a given point and
+    any previously selected points. This implicitly constructs a Voronoi
+    tessellation which is updated with each new selection, as each unselected
+    point "belongs" to the Voronoi polyhedron of the nearest previous selection.
+
+    This implicit tessellation enabled a more efficient evaluation of the FPS --
```

Comment:
My own wall-time benchmarks indicate that VoronoiFPS is always 2x to 4x times slower than the simple FPS implementation. Is this also the case in your benchmarks?

If so, this should be reflected in the documentation to make sure we manage people expectations: while VoronoiFPS can be much faster it currently is not.